src/Microsoft.OData.Extensions.Client.Abstractions/Microsoft.OData.Extensions.Client.Abstractions.Nightly.nuspec

```diff
@@ -4,8 +4,8 @@
         <id>Microsoft.OData.Extensions.Client.Abstractions</id>
         <title>Microsoft Extensions OData Client.Abstractions</title>
         <version>$VersionFullSemantic$-Nightly$NightlyBuildVersion$</version>
-        <authors>Microsoft</authors>
-        <copyright>&#169; Microsoft Corporation. All rights reserved.</copyright>
+        <authors>.NET Foundation</authors>
+        <copyright>&#169; .NET Foundation and Contributors.</copyright>
```

Comment:
@marabooy You retained the "All rights reserved." part in these other 2 .nuspec files:
- src/Microsoft.OData.Extensions.Migration/Microsoft.OData.Extensions.Migration.Nightly.nuspec
- src/Microsoft.OData.Extensions.Migration/Microsoft.OData.Extensions.Migration.Release.nuspec